FAQ.md

```diff
@@ -94,13 +94,14 @@ Unless the last release was an LTS one, you will generally want to remove one of
 
 We perform different builds such as the based openjdk (hotspot), builds from the Eclipse OpenJ9 codebase as well as others such as Corretto and SAPMachine. These alternatives are referred to as build variants.
 
-First you will need to add support into the [pipeline files](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build) as well as any environment-specific change syou need to make in the [platform files](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/build-farm/platform-specific-configurations)
```

Comment:
Thanks for fixing all of my random typos from when I knocked this up initially (I suspect they are mine) - was always intended to be a quick first pass at a FAQ but I'm a bit shocked at how many I put in :-)

Reply:
To be fair, there weren't that many and I only noticed them when I started running my spellchecker against the docs. It's fairly easy to miss them